bundles/org.openhab.core.automation/src/main/java/org/openhab/core/automation/internal/RuleEngineImpl.java

```diff
@@ -1599,7 +1601,8 @@ private void executeRulesWithStartLevel() {
                     .filter(this::mustTrigger) //
                     .forEach(r -> runNow(r.getUID(), true,
                             Map.of(SystemTriggerHandler.OUT_STARTLEVEL, StartLevelService.STARTLEVEL_RULES, "event",
-                                    SystemEventFactory.createStartlevelEvent(StartLevelService.STARTLEVEL_RULES))));
+                                    SystemEventFactory.createStartlevelEvent(StartLevelService.STARTLEVEL_RULES,
+                                            RuleEngineImpl.class.getName() + ".executeRulesWithStartLevel"))));
```

Comment:
... and here (and don't add the method name; users shouldn't know or care, as commented on the PR itself) ...